packages/enzyme/src/Utils.js

```diff
@@ -362,3 +362,15 @@ export function displayNameOfNode(node) {
 
   return type.displayName || (typeof type === 'function' ? functionName(type) : type.name || type);
 }
+
+export function sym(s) {
+  return typeof Symbol === 'function' ? Symbol.for(`enzyme.${s}`) : s;
```

Comment:
the presence of `Symbol` doesn't ensure `Symbol.for` is present - this should maybe be a fallback from `Symbol.for` to a memoized `Symbol` to ``` `___${s}___` ```?